debian/rules

```diff
@@ -59,6 +59,7 @@ override_dh_installinit:
 	install -d -m0755 debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-osd*.conf debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-disk.conf debian/ceph-osd/etc/init
+	install -m0644 build/src/upstart/ceph-osd.conf debian/ceph-osd/etc/init
```

Comment:
1. I don't think this is duplicated with line 60 because I renamed `ceph-osd.conf` to `ceph-osd.conf.in`. So `ceph-osd*.conf` won't match `ceph-osd.conf.in`.
2. If I install `ceph-osd.conf` in `debian/rules`, do I need to include it in `debian/ceph-osd.install`?
3. If I don't install `ceph-osd.conf` in `src/CMakeLists.txt`, do I need to `rm` it in `ceph.spec.in`?